.github/actions/docker-publish/action.yml

```diff
@@ -0,0 +1,73 @@
+name: "Docker Publish"
+description: "Builds and publishes a Docker image for the Ganache repo."
+inputs:
+  VERSION:
+    description: "The release version to be tagged to the Docker image, e.g. vX.x.x"
+    required: true
+  TAG:
+    description: "The tag to be tagged to the Docker image, e.g. latest."
+    required: true
+  DOCKER_USERNAME:
+    description: "Username to log into Docker hub."
+    required: true
+  DOCKER_ACCESS_TOKEN:
+    description: "Access token to log into Docker hub."
+    required: true
+  GITHUB_TOKEN:
+    description: "Github token used to log into the container registry."
+    required: true
+  INFURA_KEY:
+    description: "Infura key used to build the Docker image."
+    required: true
+runs:
+  # this means we are using yml (rather than js or docker) for our action
+  # https://docs.github.com/en/actions/creating-actions/creating-a-composite-action
+  using: "composite"
+  steps:
+    - name: Use npm 14
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14
+
+    - name: Run installation
+      run: npm ci
+      shell: bash
```

Comment:
Do we need to `npm ci` here , outside of Docker, since we also `npm ci` within Docker?

Reply:
Yes. Why? I don't know.